app/controllers/proposals_controller.rb

```diff
@@ -23,6 +23,7 @@ def index
   def new
     @proposal = Proposal.new(event: @event)
```

Comment:
Though this was already here, I prefer this style:

@proposal = @event.proposals.new